### PR TITLE
remove cryptography version pinning due to paramiko newer than 2.11.0

### PR DESCRIPTION
**Pull request by TurnOffNOD**
## Description

according to pull request #1055 and file setup.py
` # Pinning cryptography should not be needed after paramiko 2.11.0.`

Now that paramiko's version is 3.1.0, arch package python-paramiko version is 2.11.1, both newer than 2.11.0, so remove cryptography version pinning.

## Checklist

- [x] I've added this contribution to the `changelog.md`.
- [x] I've added my name to the `AUTHORS` file (or it's already there).

### CODE REVIEW

**Comment by Kreijstal** [comment]
I would like this as well

**Comment by ambrahma** [comment]
Any reason why this PR is still blocked?

File: setup.py [hunk 1]
```diff
@@ -18,9 +18,8 @@
 
 install_requirements = [
     'click >= 7.0',
-    # Temporary to suppress paramiko Blowfish warning which breaks CI.
-    # Pinning cryptography should not be needed after paramiko 2.11.0.
-    'cryptography == 36.0.2',
+    # Pinning cryptography is not needed after paramiko 2.11.0. Correct it
+    'cryptography >= 1.0.0',
```
Comment (laixintao) [inline setup.py:22]: If pinning is not needed, I think we can just delete this line, because we didn't depend on cryptography directly from code.

**Review by laixintao — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (laixintao) [reply setup.py:22]: apologies for the delay 🥺

can you resolve the conflict as well? thanks

Comment (TurnOffNOD) [reply setup.py:22]: > apologies for the delay 🥺
> 
> can you resolve the conflict as well? thanks

Do you mean the "Resolve conversation" button below?

Comment (laixintao) [reply setup.py:22]: No, I mean, this PR can not be merged now, cause it has conflicts.

image

Can you rebase from main branch, or resubmit one? thanks.

Comment (TurnOffNOD) [reply setup.py:22]: > No, I mean, this PR can not be merged now, cause it has conflicts.
> image
> 
> Can you rebase from main branch, or resubmit one? thanks.

Oh，please allow me some time.

**Comment by amjith** [comment]
Thank you!

**Pull request merged**